src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
+    result: 0,
+
+    operate(operation, num) {
+      operation(num);
+
+      return obj;
```

Comment:
The `operate` method should update the `result` property of the calculator object. However, it currently calls the operation without applying the result to `obj.result`. To fix this, you should pass `obj.result` to the operation and update it accordingly.